sg_HrtfSpatAlgorithm.cpp

```diff
@@ -53,15 +53,20 @@ HrtfSpatAlgorithm::HrtfSpatAlgorithm(SpeakerSetup const & speakerSetup,
 {
     JUCE_ASSERT_MESSAGE_THREAD;
 
+    static auto const hrtfDir{ juce::File::getCurrentWorkingDirectory().getChildFile("hrtf_compact") };
```

Comment:
I think with these changes it'll be important to check if the macOS .app hasn't broken as these always have weird paths